npm-update.js

```diff
@@ -1,189 +1,88 @@
 /* Copyright (c) 2014-2015 Richard Rodger, MIT License */
 /* jshint node:true, asi:true, eqnull:true */
-"use strict";
+"use strict"
 
 
-var follow = require('follow');
-var npm_stats = require('npm-stats')
-var async     = require('async')
+var follow     = require('follow')
+var NpmStats   = require('npm-stats')
+var async      = require('async')
+var JSONStream = require('JSONStream')
 
 
-module.exports = function npm_update( options ){
+module.exports = function npmUpdate( options ){
   var seneca = this
 
-  options = seneca.util.deepextend({
-    task: null,
-    batchsize: 33,
+  var opts = Object.assign({
```

Comment:
We need a limit option, default 1000. This is the limit that is set by default when listing. When we run in live let's use `UPDATER_LIMIT=0` to specify no limit. The use case here is to stop the updater downloading the world when testing. @Georgette You may have a better limit in mind?